src/fiatExchanges/PaymentMethodSection.tsx

```diff
@@ -105,6 +108,37 @@ export function PaymentMethodSection({
     </>
   )
 
+  const renderProviderInfo = (quote: NormalizedQuote) => (
+    <View style={styles.providerInfoContainer}>
+      <View style={styles.imageContainer}>
+        <Image
+          testID={`image-${quote.getProviderName()}`}
+          source={{ uri: quote.getProviderLogo() }}
+          style={styles.providerImage}
+          resizeMode="center"
+        />
+      </View>
+      {quote.isProviderNew() && (
+        <Touchable
+          testID={`newLabel-${quote.getProviderName()}`}
+          style={styles.newLabelContainer}
+          onPress={() => {
+            ValoraAnalytics.track(FiatExchangeEvents.cico_providers_new_info_opened, {
```

Comment:
Should we track the fiat account schema here too, since a provider could have multiple options available?

Reply:
added paymentMethod since a quote here could be FC or non FC